.github/workflows/reusable-test-core-build-process.yml

```
@@ -105,6 +108,9 @@ jobs:
       - name: Install npm Dependencies
         run: npm ci
 
+      - name: Check for PHP functions with the gutenberg_ prefix
+        run: npm run grunt prevent-gutenberg-functions || true
```

Will the `|| true` be removed?

```suggestion
        run: npm run grunt prevent-gutenberg-functions
```

---

The intention with including `|| true` here was to [avoid having to add another workflow input controlling whether the check is run or not](https://github.com/WordPress/wordpress-develop/blob/trunk/.github/workflows/reusable-test-core-build-process.yml#L19-L28). The command would fail in older branches because the task is not present, and `|| true` would just skip the failure.

But I see now that the workflow just always passes because of this. So the input is likely the only way to properly account for this.

---

I've updated the workflow to use an input that defaults to `false` instead.